submission-instructions.md

```diff
@@ -12,13 +12,15 @@ by the submission system and/or subject to an administrative reject.
     review by another conference or journal. You may submit material
     presented previously at a workshop without copyrighted proceedings.
 
--   The submission is limited to eleven (11) pages in the IEEE 8.5" x
-    11" (US letter size paper) Two-Column Format using 10pt font, with
-    no more than 6 lines per inch. This page limit applies to all
+-   The submission is limited to eleven (11) pages in the ACM 8.5" x
+    11" (US letter size paper) using 9pt font, with
+    no more than 7 lines per inch. This page limit applies to all
```

Comment:
The font size change comes from https://www.acm.org/publications/taps/word-template-workflow. This might be more material than fits onto the IEEE template. Would you like to reduce the page count as a result?

Reply:
Indeed, the ACM template allows for a lot of content. Let us make 10 pages + references. We can allow more pages for the final version.

Reply:
Done: 7790c25